src/main/java/io/openliberty/website/starter/metadata/Constants.java

```diff
@@ -10,19 +10,23 @@
  *******************************************************************************/
 package io.openliberty.website.starter.metadata;
 
-import java.util.Arrays;
 import javax.json.Json;
 import javax.json.JsonObject;
 
 public class Constants {
+
     public static final String[] SUPPORTED_JAVA_VERSIONS = new String[] { "8", "11", "17" };
-    public static final String[] SUPPORTED_JAKARTAEE_VERSIONS = new String[] { "None","7.0", "8.0", "9.1" };
-    public static final String[] SUPPORTED_MICROPROFILE_VERSIONS = new String[] { "None","1.4", "2.2", "3.3", "4.1", "5.0" };
+
+    public static final String[] SUPPORTED_JAKARTAEE_VERSIONS = new String[] { "None", "7.0", "8.0", "9.1", "10" };
+
+    public static final String[] SUPPORTED_MICROPROFILE_VERSIONS = 
+        new String[] { "None", "1.4", "2.2", "3.3", "4.1", "5.0", "6" };
```

Comment:
I have gotten feedback that there is a preference for `MicroProfile 6` over `MicroProfile 6.0`. I have also applied it to the choices for the starter despite `6` looking different from the rest of the existing choices.

Reply:
Hm, I think to be programmatically correct, we need to have it be `6.0` so that the `pom.xml` is correct. 

Right now the `pom.xml` in a Maven starter project shows `6` which is getting flagged as a "404 not found"
<img width="596" alt="image" src="https://user-images.githubusercontent.com/31117513/228358775-775ac546-42e9-4721-80f4-3d2b7f69801b.png">

Changing it to `6.0` fixes the VS Code warningig.